docker/prod/app_environment_variables.rb

```diff
@@ -4,6 +4,8 @@
 
 ENV['LABBOOK_PROVIDER_URL']              ||= 'https://labbook.concord.org'
 
+ENV['ACTIVITY_PLAYER_URL']               ||= 'https://activity-player.concord.org/branch/master'
```

Comment:
It'd be best to just remove this, it follows the pattern of not using this file, and if someone deploys LARA without a configuration will get the master branch which won't be stable.